test-configs.yaml

```diff
@@ -642,9 +647,21 @@ device_types:
     class: arm-dtb
     boot_method: uboot
 
+  imx7ulp-evk:
+    mach: imx
+    class: arm-dtb
+    boot_method: uboot
+
   imx8mn-ddr4-evk:
     mach: imx
-    class: arm64-dtb
+    arch: arm64
+    dtb: 'freescale/imx8mn-ddr4-evk.dtb'
+    boot_method: uboot
+
+  imx8mp-evk:
```

Comment:
This one has run but it failed to reach a login prompt, it seemed stuck waiting for the network interface:
https://staging.kernelci.org/test/plan/id/5f32c8abe9e0d5d52e47b0e3/
https://storage.staging.kernelci.org/kernelci/staging.kernelci.org/staging-20200811.2/arm64/defconfig/gcc-8/lab-nxp/baseline-imx8mp-evk.html

Reply:
Tested OK on staging now: https://staging.kernelci.org/test/plan/id/5f4f57b549d566666d39b1e8/